src/c++/uds/src/uds/logger.h

```diff
@@ -37,11 +38,8 @@
 #endif /* __KERNEL__ */
 
 #ifdef __KERNEL__
-#if defined(MODULE)
-#define UDS_LOGGING_MODULE_NAME THIS_MODULE->name
-#else /* compiled into the kernel */
-#define UDS_LOGGING_MODULE_NAME "vdo"
-#endif
+#define DM_MSG_PREFIX "vdo"
+#define UDS_LOGGING_MODULE_NAME DM_NAME ": " DM_MSG_PREFIX
 #else /* userspace */
```

Comment:
I would like to keep the #if defined(MODULE) clause, inside an #ifndef VDO_UPSTREAM, in case some of our perl tests rely on the logging that produces.  This can be cone in this commit (the second), or it can be done with a follow-up internal only patch. But I'd prefer not to lose it.

Reply:
I think this is a bad reason to keep this for internal. We should use and test what the kernel uses. If tests break, we should fix the tests.